src/services/service-line-template/index.html

```diff
@@ -0,0 +1,356 @@
+        {% assign text = site.translations[site.lang].software-modernisation.hero-cta %}
+        {% include cta_primary_link.html color='coral' href=href text=text %}
+      </div>
+    </div>
+    <div class="sm-hero__grid__grid-item sm-hero__grid__grid-item--intro-image" style="background-image: url({{ '/assets/custom/img/placeholders/meta-social-image.png' | prepend: site.baseurl }})">
+      <img class="img-parallax" src="{{ '/assets/custom/img/placeholders/meta-social-image.png' | prepend: site.baseurl }}">
+    </div>
+  </div><!-- grid -->
+</section><!-- sm-hero -->
+
+
+<section class="soft-mod-problems">
+  {%- assign anchor_id = "how-we-solve-your-problems" -%}
+  {% include page_location_anchor.html anchor_id=anchor_id %}
+
+  {% capture title %}{% t service-line-template.problems-title %}{% endcapture %}
+  {% capture description %}{% t service-line-template.problems-introduction %}{% endcapture %}
+  {% assign cta_href = "#get-in-touch" %}
+  {% capture cta_text %}{% t software-modernisation.hero-cta %}{% endcapture %}
+  {% include section_intro.html class="soft-mod-problems__intro" cta_href=cta_href cta_text=cta_text description=description title=title %}
+
+  <div class="container z-index-1">
+    <div class="tabbed-content g-pt-50">
+      <nav class="tabs">
+        <ul>
+          <li><a href="#sustainable-change" class="active">{% t software-modernisation.sustainable-change-title %}</a></li>
+          <li><a class="g-pt-15" href="#innovation">{% t software-modernisation.innovation-title %}</a></li>
+          <li><a class="g-pt-15"href="#technology">{% t software-modernisation.technology-title %}</a></li>
+          <li><a href="#business-alignment">{% t software-modernisation.business-alignment-title %}</a></li>
+          <li><a href="#people-culture">{% t software-modernisation.people-and-culture-title %}</a></li>
+          <li><a href="#risk-management">{% t software-modernisation.risk-management-title %}</a></li>
+        </ul>
+      </nav><!-- tabs -->
+
+      <article id="sustainable-change" class="item active" data-title="Sustainable Change">
+        <div class="item-content">
+          <div class="item-content-grid">
+            <div class="item-content-grid__item-left bg-blue bg-sustainable-change">
+              <span class="h2 text-900 text-white">{% t software-modernisation.sustainable-change-intro %}</span>
+            </div>
+            <div class="item-content-grid__item-right bg-white g-pt-35">
+              <span class="h4 text-700">{% t software-modernisation.what %}</span>
+              <ul class="g-pb-15 list-ticks">
+                <li>{% t software-modernisation.sus-change-li-1 %}</li>
+                <li>{% t software-modernisation.sus-change-li-2 %}</li>
+                <li>{% t software-modernisation.sus-change-li-3 %}</li>
+                <li>{% t software-modernisation.sus-change-li-4 %}</li>
+                <li>{% t software-modernisation.sus-change-li-5 %}</li>
```

Comment:
Not on this branch necessarily, but let's move the shared translation strings out of "software-modernisation"?